pkg/insight/insightstore/deploymentstore.go

```diff
@@ -0,0 +1,42 @@
+// Copyright 2022 The PipeCD Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package insightstore
+
+import (
+	"context"
+	"errors"
+
+	"github.com/pipe-cd/pipecd/pkg/model"
+)
+
+var (
+	errUnimplemented = errors.New("unimplemented")
+)
+
+type DeploymentStore interface {
+	// GetDeployments returns slice of Deployment sorted by startedAt ASC.
+	GetDeployments(ctx context.Context, projectID string, from, to int64, minimumVersion *model.InsightDeploymentVersion) ([]*model.InsightDeployment, error)
```

Comment:
I think it better this interface name is `List` because it's not acquired by PK.

Reply:
Ops, It's too late lol

Reply:
I'll fix it on another PR.